### DISCUSSION TITLE
Count votes on Status Checks

### DISCUSSION THREAD

**Discussion by sshiells-scottlogic** [cncf/gitvote#538] [Ideas]
We have just started using gitvote and we dedcied to include a daily status update to keep track of things. However this confused the voters and people started to vote on the status checks instead of the actual vote. I was wondering what people think of possibly counting the votes on the status checks toward the total?

**Comment by tegioz** [comment]
Hi @sshiells-scottlogic 👋

IMHO allowing users to vote on multiple comments could be confusing and lead to problems (i.e. one of those status checks comments is removed after someone has reacted/voted on it). This would also mean adding some more complexity to GitVote, and would also require additional API calls to collect reactions from potentially multiple comments, specially for votes that remain open for a long time. We try to be really careful with any additional API call introduced to avoid hitting any rate limits.

Maybe we could make this clearer in the status check comment, adding some sort of note that stands out to let users know the right place to vote.

**Reply by sshiells-scottlogic** [reply]
Yeah the more I think about it I would agree that it would be a bit messier. I think adding a note to make it clearer might be nice.

One other thing that might make sense, though I am not sure if it would work, but would the status checks be able to edit the vote comment and addthe current status to that (not even sure if that would be possible).

**Reply by tegioz** [reply]
We discussed that some time ago, please see the comments in https://github.com/cncf/gitvote/issues/162.

We'll add that note then 🙂

**Reply by sshiells-scottlogic** [reply]
Ah cool. Tbh I think better user training is all that is likely to be required to fix our issues, but i was asked to explore the options so here we are.

**Reply by tegioz** [reply]
Sure, no problem. Feel free to ask any question if something isn't clear or you encounter any problem!
